app/cdap/components/ResourceCenterEntity/ResourceCenterPipelineEntity.scss

```diff
@@ -30,7 +30,7 @@ $button-width: $height-of-image;
 
     .buttons-container {
       position: absolute;
-      bottom: 8px;
+      bottom: 2.5px;
```

Comment:
2.5px?? does it need to be that exact??

Reply:
yea the pipelines card button container is different from other cards button containers. I had to adjust like this to make them look align horizontally.